docs/development/test.html

```diff
@@ -19,7 +19,7 @@
         }
     </script>
     <![endif]-->
-
+    <script src="https://unpkg.com/mocha/mocha.js"></script>
```

Comment:
This means that the testing repo has an external dependency off of Microsoft's network; this generates unnecessary traffic and slows your builds down ~5 seconds.  

If you install mocha as a dev dependency instead, you will get these from Microsoft's internal package re-host, which is much much faster.

It also means that as Microsoft begins to engage security on their package re-host, you aren't left out.

Reply:
I'll take a patch for that once this drops.  I'd rather spend time moving to whatever we pick that is not-mocha.

Reply:
Start a new issue for this, please, if you want to track it.  Otherwise, resolve?

Reply:
these are comments, not reviews, so i think that they can't be resolved

i could be wrong

[issue is here](https://github.com/peggyjs/peggy/issues/78)

Reply:
Ah, I might have to be the one to resolve it.